testdata/p4_16_samples/parenthesis-test_ubpf.p4

```diff
@@ -0,0 +1,107 @@
+/*
```

Comment:
this test in itself is not useful, because we do not check the resulting C file.
Ideally you should have an stf file to check that the evaluated expression has the correct semantics.